Assets/Scripts/UI/CombatHUD/CombatTutorial.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```

Comment:
Don't have to, but it would be nice to have a summary comment here explaining what the class is for and where it is used.

Reply:
Shit, forgot about that, have added one!